ckanext/data_qld_theme/templates/package/snippets/resources.html

```diff
@@ -1,11 +0,0 @@
-{% ckan_extends %}
-
-{% block resources_list %}
-    <ul class="list-unstyled nav nav-simple">
-    {% for resource in resources %}
```

Comment:
where is this being handled now

Reply:
It's in core. Our override didn't actually add value.